docs/src/main/asciidoc/security-basic-authentication-tutorial.adoc

```diff
@@ -14,15 +14,15 @@ include::{includes}/prerequisites.adoc[]
 
 == What you will build
 
-The steps in this tutorial guide you through building an application that provides the following endpoints:
+The steps in this tutorial guide you through building an application that provides the following endpoints to demonstrate different authorization policies:
 
 [cols="20%,40% ",options="header"]
 |===
 |Endpoint | Description 
 |`/api/public`| The `/api/public` endpoint can be accessed anonymously.
 |`/api/admin`| The `/api/admin` endpoint is protected with role-based access control (RBAC), and only users who have been granted the `admin` role can access it.
-At this endpoint, the `@RolesAllowed` annotation is used to declaratively enforce the access constraint.
-|`/api/users/me`| The `/api/users/me` endpoint is protected with RBAC and only users that have been granted the `user` role can access it. A JSON document with details about the user is returned as a response.
+At this endpoint, the `@RolesAllowed` annotation is used to enforce the access constraint declaratively.
+|`/api/users/me`| The `/api/users/me` endpoint is protected with RBAC and only users that have been granted the `user` role can access it.
```

Comment:
You're missing the sentence about what it returns (it should be just a string with the username - not a JSON document)

Reply:
Yeah, I just add it. THX :)